src/sass/_basics.scss

```diff
@@ -1,10 +1,17 @@
 $include-html: false !default;
 
 @if ($include-html) {
+  html {
+    font-size: $baseFont;
```

Comment:
By changing this value we can tweak the sizes of **every** component in style-guide.
Use with case!
Relates to #108